mir-module/src/main/resources/xslt/generate-svg-by-type.xsl

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="3.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+  <xsl:output method="xml"
+              media-type="image/svg+xml"
+              doctype-public="-//W3C//DTD SVG 1.1//EN"
+              doctype-system="http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"/>
+
+  <xsl:param name="extension" select="'?'"/>
+
+  <xsl:template match="*">
+
+    <xsl:variable name="value">
+      <xsl:choose>
+        <xsl:when test="string-length($extension) &lt; 5">
+          <xsl:value-of select="$extension"/>
+        </xsl:when>
+        <xsl:otherwise>
+          <xsl:value-of select="substring($extension, 0, 4)"/>
+        </xsl:otherwise>
+      </xsl:choose>
+    </xsl:variable>
+
+    <xsl:call-template name="generate-download-svg">
+      <xsl:with-param name="type" select="$value"/>
+    </xsl:call-template>
+  </xsl:template>
+
+  <xsl:template name="generate-download-svg">
+    <xsl:param name="type"/>
+
+    <svg id="Ebene_1" data-name="Ebene 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 36.88 41">
```

Comment:
Attributes `id` and `data-name` are unnecessary. Probably just a remnant from whichever editor was used to create the original SVG.